assets/scripts/components/grouped-item-listings.js

```diff
@@ -116,11 +116,12 @@ export function initializeGroupedListings() {
         const activeCategory = document.querySelector('.controls .active');
         const activeCategoryFilter = (activeCategory) ? activeCategory.dataset.filter : '';
 
-        if(searchValue){
+        if (searchValue) {
             url.searchParams.set('search', searchValue)
-        }else{
+        } else {
             url.searchParams.delete('search')
         }
+
```

Comment:
@bgdeutsch the search functionality is not working the same way. on live, you can search by the  action name ([data-name]). something probably needs to be updated here

Reply:
it could also be something int he actioncatalog/list.html

Reply:
Just confirming that we'd like the search to work for action names

Reply:
@hestonhoffman @StefonSimmons this is fixed thank you both for flagging!

Reply:
Should this be fixed in the staging preview as well? I still see the issue there